packages/Modal/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@paprika/modal",
+  "version": "0.1.0",
+  "description": "Modal component",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Modal"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "directories": {
```

Comment:
Since this doesn't seem to be doing anything useful yet (https://docs.npmjs.com/files/package.json#directories) I'd omit for now? Otherwise there may be confusion from people seeing this and going "hmm what's this do? this is probably doing something useful... why is it only in this package but not others though? is this one different somehow?"

Reply:
This lines automatically generated by `npx lerna create ...`. I don't think that it is good idea to remove them every time we start new package.

Reply:
I guess that's fair (although IMO we should feel free to change lerna's output if it doesn't fit our needs), but my point still stands. Not blocking though

This also brings up the point of having a blessed way to create new packages, e.g. I think @nahumzs had a script for that in the beginning.

Reply:
@DanilAgafonov this will be removed from Lerna
![2B1F6FEA-1E84-4CED-9A8B-AFF58E240580](https://user-images.githubusercontent.com/93752/70682165-c23e9a00-1c52-11ea-82db-a2c79354d2ad.jpeg)

Reply:
Oh, thanks. I removed `"directories"` in this commit: https://github.com/acl-services/paprika/pull/320/commits/e728bbe1781d14142130b9267660c5d712a604d2

Reply:
Yes, we had a script at the beginning we later remove it, we can bring it back if we think will provide value.